addons/escoria-core/game/inputs_manager.gd

```diff
@@ -31,7 +31,19 @@ var hover_stack: Array = []
 # The global id of the topmost item from the hover_stack
 var hotspot_focused: String = ""
 
-var custom_background_input_handler = null
+# Function that can be used to intercept and process input events.
```

Comment:
```suggestion
# Function reference that can be used to intercept and process input events.
```

Just to make it painfully clear that `var custom_input_handler` isn't actually a function implementation (call me pedantic).

Reply:
I added "reference" here and in one other docstring.